include/deal.II/lac/la_vector.h

```diff
@@ -110,81 +110,88 @@ namespace LinearAlgebra
     /**
      * Multiply the entire vector by a fixed factor.
      */
-    virtual VectorSpaceVector<Number> &operator*= (const Number factor);
+    virtual Vector<Number> &operator*= (const Number factor) override;
```

Comment:
Right now I am using C++11 because it's easier but if we agree on the new interface, I will get ride of it.